tfexec/workspace_show.go

```diff
@@ -0,0 +1,21 @@
+package tfexec
+
+import (
+	"context"
+	"strings"
+)
+
+// WorkspaceShow represents the workspace show subcommand to the Terraform CLI.
+func (tf *Terraform) WorkspaceShow(ctx context.Context) (string, error) {
+	workspaceShowCmd := tf.buildTerraformCmd(ctx, nil, "workspace", "show", "-no-color")
+
+	var outBuffer strings.Builder
+	workspaceShowCmd.Stdout = &outBuffer
+
+	err := tf.runTerraformCmd(ctx, workspaceShowCmd)
+	if err != nil {
+		return "", err
+	}
+
+	return strings.TrimSpace(outBuffer.String()), nil
+}
```

Comment:
i didn't break this up into `WorkspaceShow` and `workspaceShowCmd` because it didn't seem like that would be very helpful here since this command doesn't really have options i don't think. i'm happy to break it out that way for consistency or if i've missed something though. just let me know!